pubspec.yaml

```diff
@@ -3,7 +3,7 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 9.0.1
+version: 10.0.0-dev.0
 description: Generator for FFI bindings, using LibClang to parse C header files.
```

Comment:
This should be updated to "Generator for FFI bindings, using LibClang to parse C, Objective-C and Swift files."

It's probably fair enough to say that we support Objective-C and Swift now.